File: lib/models/command/command_model.dart
class Command {
  String? cmdId;
  String? cmdCode;
  String? commitTime;
  String? content;
  String? isImme;
  String? returnTime;
  String? returnValue;
  String? sn;
  String? sync;
  String? done;
  String? secondWait;

  Command({
    this.cmdId,
    this.cmdCode,
    this.commitTime,
    this.content,
    this.isImme,
    this.returnTime,
    this.returnValue,
    this.sn,
    this.sync,
    this.done,
    this.secondWait,
  });

  factory Command.fromJson(Map<String, dynamic> json) {
    return Command(
      cmdId: json['cmd_id'],
      cmdCode: json['cmd_code'],
      commitTime: json['commit_time'],
      content: json['content'],
      isImme: json['is_imme'],
      returnTime: json['return_time'],
      returnValue: json['return_value'],
      sn: json['sn'],
      sync: json['sync'],
      done: json['done'],
      secondWait: json['second_wait'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'cmd_id': cmdId,
      'cmd_code': cmdCode,
      'commit_time': commitTime,
      'content': content,
      'is_imme': isImme,
      'return_time': returnTime,
      'return_value': returnValue,
      'sn': sn,
      'sync': sync,
      'done': done,
      'second_wait': secondWait,
    };
  }
}

File: android/app/src/main/kotlin/com/example/play_box/model/command/CommandModel.kt
package com.example.play_box.model.command

import com.google.gson.annotations.SerializedName

data class CommandModel(
    @SerializedName("cmd_id")
    val cmdId: String? = null,
    @SerializedName("cmd_code")
    val cmdCode: String? = null,
    @SerializedName("commit_time")
    val commitTime: String? = null,
    val content: String? = null,
    @SerializedName("is_imme")
    val isImme: String? = null,
    @SerializedName("return_time")
    val returnTime: String? = null,
    @SerializedName("return_value")
    val returnValue: String? = null,
    val sn: String? = null,
    val sync: String? = null,
    val done: String? = null,
    @SerializedName("second_wait")
    val secondWait: String? = null,
)